include/aspect/postprocess/visualization.h

```diff
@@ -124,6 +124,20 @@ namespace aspect
       class Interface
       {
         public:
+          /**
+           * Constructor. The constructor takes as argument the physical
+           * units of the quantity (scalar or vector-valued) computed by
+           * derived classes. The empty string, "", refers to an unknown
+           * or nonexistent unit.
+           *
+           * If a visualization postprocessor generates more than one
+           * output component, and if the different components have different
+           * physical units, then they should be separated by commas. If
+           * the different components have the same physical units, these units
+           * need to be specified only once and will apply to all components.
+           */
+          Interface (const std::string &physical_units = "");
```

Comment:
```suggestion
          explicit Interface (const std::string &physical_units = "");
```

Reply:
That is true, but I flag any one-argument constructors in reviews without thinking about these details. :-)